CMake/SlicerBlockFindQtAndCheckVersion.cmake

```diff
@@ -72,16 +85,23 @@ if(NOT Slicer_SOURCE_DIR)
 endif()
 message(STATUS "Configuring ${_project_name} with Qt ${_major}.${_minor}.${_patch} (using modules: ${command_separated_module_list})")
 
-# Since Qt5 does not set CMake variables for plugins and binary directories,
+# Since neither Qt5 or Qt6 set CMake variables for plugins and binary directories,
 # we explicitly set them here.
-  set(QT_PLUGINS_DIR "${Qt5_DIR}/../../../plugins")
-  get_filename_component(QT_PLUGINS_DIR ${QT_PLUGINS_DIR} ABSOLUTE)
-  message(STATUS "Setting QT_PLUGINS_DIR: ${QT_PLUGINS_DIR}")
+set(QT_PLUGINS_DIR "${Qt${_major}_DIR}/../../../plugins")
```

Comment:
`Qt6_DIR=/usr/lib/cmake/Qt6`, and the plugins dir is `/usr/lib/qt6/plugins` in ArchLinux. but here `${Qt${_major}_DIR}/../../../plugins=/usr/plugins`